assignment_drafts.feature

```diff
@@ -1,4 +1,4 @@
-@plugin @plagiarism @plagiarism_turnitinsim @plagiarism_turnitinsim_assignment @plagiarism_turnitinsim_assignment_drafts
+@plugin @plagiarism  @plagiarism_turnitinsim_assignment @plagiarism_turnitinsim_assignment_drafts
```

Comment:
Do we not want this as part of the whole suite?

Reply:
I agree, all tests should have the @plagiarism_turnitinsim tag so we can run the full suite.